SuperBuild/External_LZMA.cmake

```diff
@@ -8,27 +8,32 @@ set(${proj}_DEPENDENCIES "")
 ExternalProject_Include_Dependencies(${proj} PROJECT_VAR proj DEPENDS_VAR ${proj}_DEPENDENCIES)
 
 if(Slicer_USE_SYSTEM_${proj})
-  unset(${proj}_DIR CACHE)
-  unset(${proj} CACHE)
-  find_package(${proj} REQUIRED)
+  unset(LZMA_INCLUDE_DIR CACHE)
+  unset(LZMA_LIBRARY CACHE)
+  find_path(LZMA_INCLUDE_PATH lzma.h)
+  find_library(LZMA_LIBRARY lzma)
 endif()
 
 # Sanity checks
-if(DEFINED ${proj}_DIR AND NOT EXISTS ${${proj}_DIR})
-  message(FATAL_ERROR "${proj}_DIR variable is defined but corresponds to nonexistent directory")
+if(DEFINED LZMA_INCLUDE_DIR AND NOT EXISTS ${LZMA_INCLUDE_DIR})
+  message(FATAL_ERROR "LZMA_INCLUDE_DIR variable is defined but corresponds to nonexistent directory")
+endif()
+if(DEFINED LZMA_LIBRARY AND NOT EXISTS ${LZMA_LIBRARY})
+  message(FATAL_ERROR "LZMA_LIBRARY variable is defined but corresponds to nonexistent directory")
 endif()
 
-if(NOT DEFINED ${proj}_DIR AND NOT Slicer_USE_SYSTEM_${proj})
+if((NOT DEFINED LZMA_INCLUDE_DIR
+   OR NOT DEFINED LZMA_LIBRARY) AND NOT Slicer_USE_SYSTEM_${proj})
 
   ExternalProject_SetIfNotDefined(
     Slicer_${proj}_GIT_REPOSITORY
-    "${EP_GIT_PROTOCOL}://github.com/nextgis-borsch/lib_lzma.git"
+    "${EP_GIT_PROTOCOL}://github.com/Slicer/lib_lzma.git"
     QUIET
     )
 
   ExternalProject_SetIfNotDefined(
     Slicer_${proj}_GIT_TAG
-    "v5.2.2"
```

Comment:
According to Python-3.6.7\PCbuild\readme.txt, lzma v5.2.2 is supposed to be used.

Reply:
Great. I will update the branch then